### PR TITLE
Removed useless strtolower calls

### PR DESCRIPTION
**Pull request by Frago9876543210**
## Introduction
This PR removes useless strtolower calls, because it is will called earlier.
https://github.com/pmmp/PocketMine-MP/blob/078a2486b5e56fb2e9b4f709d566b255872b80db/src/pocketmine/Server.php#L728-L737

### CODE REVIEW

File: src/pocketmine/OfflinePlayer.php [hunk 1]
```diff
@@ -44,7 +44,7 @@ class OfflinePlayer implements IPlayer, Metadatable{
 	public function __construct(Server $server, string $name){
 		$this->server = $server;
 		$this->name = $name;
-		if(file_exists($this->server->getDataPath() . "players/" . strtolower($this->getName()) . ".dat")){
+		if(file_exists($this->server->getDataPath() . "players/" . $this->name . ".dat")){
```
Comment (dktapps) [inline src/pocketmine/OfflinePlayer.php]: It's possible that an `OfflinePlayer` instance may be constructed by something else (such as a unit test), in which case the name might not be passed in lowercase. The class should not rely on the caller to do its work for it. At the least this check should be `strtolower()`d.

**Review by dktapps — CHANGES_REQUESTED** [review]
Looks OK apart from one nitpick.

File: src/pocketmine/OfflinePlayer.php [hunk 2]
```diff
@@ -43,7 +43,7 @@ class OfflinePlayer implements IPlayer, Metadatable{
 	 */
 	public function __construct(Server $server, string $name){
 		$this->server = $server;
-		$this->name = $name;
+		$this->name = strtolower($name);
```
Comment (dktapps) [inline src/pocketmine/OfflinePlayer.php]: I was talking about the `file_exists()` check. The case of the passed name should be preserved on the property.

File: src/pocketmine/OfflinePlayer.php [hunk 3]
```diff
@@ -44,7 +46,8 @@ class OfflinePlayer implements IPlayer, Metadatable{
 	public function __construct(Server $server, string $name){
 		$this->server = $server;
 		$this->name = $name;
-		if(file_exists($this->server->getDataPath() . "players/" . strtolower($this->getName()) . ".dat")){
+		$this->iusername = strtolower($this->name);
```
Comment (jasonwynn10) [inline src/pocketmine/OfflinePlayer.php]: use `$name` here instead of `$this->name`

**Review by jasonw4331 — CHANGES_REQUESTED** [review]
Here's my nitpick

**Review by jasonw4331 — APPROVED** [review]

**Comment by lukeeey** [comment]
i thought i already fixed this in my PR?

**Comment by Frago9876543210** [comment]
BTW, the problem is not just strtolower calls. Look at the that code:
```php
public function getMoney(IPlayer $player) : int{
	return (int) $this->money->get(strtolower($player->getName()), self::DEFAULT_MONEY);
}
```
Sometimes, it is quite convenient to use IPlayer as argument. However, if the IPlayer will be OfflinePlayer it return name in lower case, but if the player is online, then his name in will be returned without lower case. This PR fixes this. And adds to the interface `getLowerCaseName()`

**Pull request closed without merging**